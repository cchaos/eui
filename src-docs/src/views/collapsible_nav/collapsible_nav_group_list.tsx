import React from 'react';

import { EuiCollapsibleNavGroupList } from '../../../../src/components/collapsible_nav';

export default () => <EuiCollapsibleNavGroupList />;
