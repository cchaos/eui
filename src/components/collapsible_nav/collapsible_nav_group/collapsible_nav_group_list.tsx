import React, { HTMLAttributes, FunctionComponent } from 'react';
import { CommonProps } from '../../common';
import classNames from 'classnames';

export type EuiCollapsibleNavGroupListProps = HTMLAttributes<HTMLDivElement> &
  CommonProps & {};

export const EuiCollapsibleNavGroupList: FunctionComponent<
  EuiCollapsibleNavGroupListProps
> = ({ children, className, ...rest }) => {
  const classes = classNames('euiCollapsibleNavGroupList', className);

  return (
    <div className={classes} {...rest}>
      {children}
    </div>
  );
};
