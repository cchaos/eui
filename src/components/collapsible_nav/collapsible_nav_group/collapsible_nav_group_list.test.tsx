import React from 'react';
import { render } from 'enzyme';
import { requiredProps } from '../../../test/required_props';

import { EuiCollapsibleNavGroupList } from './collapsible_nav_group_list';

describe('EuiCollapsibleNavGroupList', () => {
  test('is rendered', () => {
    const component = render(<EuiCollapsibleNavGroupList {...requiredProps} />);

    expect(component).toMatchSnapshot();
  });
});
