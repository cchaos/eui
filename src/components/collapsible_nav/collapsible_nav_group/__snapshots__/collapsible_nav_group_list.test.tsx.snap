// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EuiCollapsibleNavGroupList is rendered 1`] = `
<div
  aria-label="aria-label"
  class="euiCollapsibleNavGroupList testClass1 testClass2"
  data-test-subj="test subject string"
/>
`;
